packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -216,25 +217,28 @@ const SceneView = ({
               }
             >
               {header !== undefined && headerShown !== false ? (
-                <View
-                  onLayout={(e) => {
-                    setCustomHeaderHeight(e.nativeEvent.layout.height);
-                  }}
-                >
-                  {header({
-                    back: previousDescriptor
-                      ? {
-                          title: getHeaderTitle(
-                            previousDescriptor.options,
-                            previousDescriptor.route.name
-                          ),
-                        }
-                      : undefined,
-                    options,
-                    route,
-                    navigation,
-                  })}
-                </View>
+                <>
+                  <ScreenStackHeaderConfig hidden />
```

Comment:
Don't we need to pass the options too?

Reply:
Do you mean other options for the <ScreenStackHeaderConfig /> or something else?